### PR TITLE
infra(debug) bump version to patch security vulnerability

### PR DESCRIPTION
**Pull request by EugeneHlushko**
**Summary**
In order to patch the security vulnerability, i had updated debug to `^2.6.9` by running `yarn upgrade debug@^2.6.9`

Screen Shot 2019-03-14 at 11 50 15 AM

**Result**
Result is in diff.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -5,91 +5,109 @@
 JSON2@0.1.0:
   version "0.1.0"
   resolved "https://registry.yarnpkg.com/JSON2/-/JSON2-0.1.0.tgz#8d7493040a63d5835af75f47decb83ab6c8c0790"
+  integrity sha1-jXSTBApj1YNa919H3suDq2yMB5A=
```
Comment (Haroenv) [inline yarn.lock]: can you undo this change to the integrity? We still use the older version of Yarn here. Thanks!

**Comment by EugeneHlushko** [comment]
Removed via: `sed -i '' '/integrity /d' ./yarn.lock`

**Comment by Haroenv** [comment]
Thank you! I'll release a patch for this ASAP

**Comment by EugeneHlushko** [comment]
Thank you! webpack/webpack.js.org will stop alerting me :)
